doc/contributors-guide/modules/ROOT/pages/coder/releasing-plugin.adoc

```diff
@@ -25,7 +25,7 @@ See _xref::user/add-eap-repository-to-ide.adoc[]_ for more information.
 . Go to https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases[GitHub releases for this plugin]
 . Edit the pre-release and uncheck the checkbox "`This is a pre-release`". +
 A GitHub action will then publish the release.
-You don't need to wait for the publishing process of the preview version to complete to be able to trigger publishing a stable version.
+It isn't needed to wait for the publishing process of the preview version to complete to be able to trigger publishing a stable version.
```

Comment:
Your change is valid, and an improvement to the original text. 

Reading the old and new text, I found that the negation (that was there before you change the text) is hard to read, and rewrote it a bit. The sentence is now also a bit shorter, what should make it easier to understand. What do you think?

```suggestion
Publishing of the stable version can start while the GitHub action publishing the the preview version is still running.
```

Reply:
Your change sounds better to me, but there is a double the that would need to be removed tho.